--- src/core/InputEngine.cpp.str	2009-11-06 03:02:22.000000000 +0100
+++ src/core/InputEngine.cpp	2009-11-06 03:02:36.000000000 +0100
@@ -1519,7 +1519,7 @@
 				exists = true;
 				QString errorstring = QString("InputEngine:: keyfact with: type=%1, key1='%2', key2='%3' already exists!\n"
 						"You should only define keyfact types one time!!\n").arg(keyFactType).arg(key1).arg(key2); 
-				printf(QS_C(errorstring));
+				printf("%s",QS_C(errorstring));
 				info().warning(errorstring);
 				break;
 			}
